esrally/rally.py

```diff
@@ -23,6 +23,7 @@
 import sys
 import time
 import uuid
+import signal
```

Comment:
The linter in CI raised an error that this import is unused:

```
12:20:20 esrally/rally.py:26:0: W0611: Unused import signal (unused-import)
```

Can you please remove it?